### DISCUSSION TITLE
Can not upload files in repository

### DISCUSSION THREAD

**Discussion by seevans** [community/community#56319] [New to GitHub] [answered]
Hello,
I only started working with GitHub and at the moment I can only upload files with no content into the repository, else I get this error message:
 'Something went really wrong, and we can't process that file'. 
Can someone tell me what this means and what todo? 
Thank you very much

**Comment by gekkowrld** [comment, marked as answer]
Refresh your browser and or check https://www.githubstatus.com/ to see if Github is online. Sometimes it helps to wait for a while and then try again. You can use Git to upload files if you have it in your system.

**Reply by Architdevpro** [reply]
Still not working after 2-3 days, I've tried all possibilities

**Reply by 0xkai0** [reply]
same problem here. any Idea of what should I do?

**Comment by usmanv10** [comment]
I have the same problem and I've followed the steps above but I'm still getting the error "Something went really wrong, and we can’t process that file.". Any other ideas?

**Comment by usmanv10** [comment]
I tried to upload a single file which is 11.4 kb but I still got the error.

**Reply by Architdevpro** [reply]
Same! I am also uploading a file of size 1.24kb and it gives me that error,
I've tried all solutions but none are working.
File name is 
EduAI-logo.png and even some others like
Bethany-convent-sample-papers.pdf of 1 mb can't upload
I've changed their names many times to see if that fixes it, I've also tried other possibilities but none are working

**Comment by manukumar7** [comment]
This error usually means there's an issue with the file you're trying to upload. Here are a few common causes and how to fix them:

🚫 Error Meaning: The file couldn’t be processed due to one of the following:

📦 File too large – Web upload limit is ~25–100 MB. Use smaller files or Git LFS.

🧾 Unsupported format – Check if the file is valid and not corrupted.

🌐 GitHub glitch – Refresh and try uploading again.

🧹 Browser issue – Clear cache or switch browsers.

💻 Better method: Use Git locally to push files—more reliable for large or multiple uploads.

**Comment by usmanv10** [comment]
I was uploading a file I had previously uploaded but with changes. It's still below 25mb and I currently have the previous file uploaded on a repository. Not sure why it suddenly stop working. This seems to be the case for all my files and all of them are below 25mb.

I have tried it on Chrome and FF browsers

I've also noticed the avators logos are broken. Not sure if related to this

**Reply by usmanv10** [reply]
Any update to this?

**Comment by GaragaNaveenKumar** [comment]
Sometimes, browser extensions or an unstable connection may cause failed uploads.

Try a different browser (e.g., Chrome, Firefox)

Disable extensions temporarily

Clear cache or try incognito mode

**Reply by Architdevpro** [reply]
I am using chrome and also used the app but none working

**Comment by hiunknown10000** [comment]
The same problem occurs with me when I started. I use github desktop ( https://desktop.github.com/download/ ) to push the entire code. At first download the GitHub desktop in your pc/laptop and login your account, then create a folder and open it via your code editor. Simply write the code and save it, at last come back github desktop and refresh it, now push your code. Go to browser and you can see your repository.

**Reply by usmanv10** [reply]
I'm not able to download Github Desktop. "This site can’t provide a secure connection. desktop.githubusercontent.com sent an invalid response. ERR_SSL_PROTOCOL_ERROR"

**Reply by hiunknown10000** [reply]
Then you can also search on your browser :- GitHub desktop download

**Reply by usmanv10** [reply]
That's where I tried to download it from.

**Reply by hiunknown10000** [reply]
Ok then I will share you the file, tell me your pc/laptop configuration.

**Comment by usmanv10** [comment]
I noticed I am able create a new file and paste code and save it. Not sure why uploading is not working

**Reply by Architdevpro** [reply]
Same!

**Comment by Architdevpro** [comment]
Can someone help me. I'm facing the same error!

**Comment by hballin** [comment]
I'm running into the same issue. I assume Github is not reliable in this case.
I experienced the failure picture that it works sometimes, but mostly not.
Now my article stands somewhere in between - it is incomplete. 😒

**Reply by saiindiaai** [reply]
You'll have to use git pull or git push for it, I had to switch to it completely

**Reply by hballin** [reply]
I'll try.

**Reply by hballin** [reply]
I used GitHub CLI, establish connection and repository, but it did fail in fact of experience on my end. 
Well Okay, I could bypass the problem by using an other webbrowser. I changed from EDGE to Chrome and had success. 
After I rethink my work, it was falling out of the sky, that GitHub is a subbrunch of Microsoft and it works even better with Google's Chrome. 🙊🙉🙈

**Reply by saiindiaai** [reply]
Oh, Great, for me it Failed on all 😭 tho I haven't used it for many months now, maybe it might be working or not 
Bye 👋🏻 
Have a nice day ahead 😊
